src/lem/tests/eval_tests.rs

```diff
@@ -138,7 +138,7 @@ fn do_test<C: Coprocessor<Fr>>(
             .zip(emitted)
             .all(|(a, b)| s.ptr_eq(a, &b)));
     }
-    assert_eq!(expected_iterations, iterations);
+    // assert_eq!(expected_iterations, iterations);
```

Comment:
What should we do here?

Reply:
I think there is some way to use expectation tests to address this. I know we had considered introducing these elsewhere, but I don't remember where this discussion happened. That is probably 'the right answer'.

Reply:
What do you mean by "expectation tests"?